sandpack-react/src/components/CodeEditor/CodeMirror.tsx

```diff
@@ -197,11 +197,19 @@ export const CodeMirror = React.forwardRef<CodeMirrorRef, CodeMirrorProps>(
         const customCommandsKeymap: KeyBinding[] = [
           {
             key: "Tab",
-            run: insertTab,
+            run: ({ state, dispatch }): boolean => {
+              indentMore({ state, dispatch });
+
+              return false;
```

Comment:
As `indentLess` returns true, this was preventing running the next command. So by returning false, it's possible to run multiple commands on the key press. 

I'm not absolutely sure about this change, but on my tests, it didn't introduce any regression,